modules/nw-ovn-kubernetes-list-resources.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="nw-list-all-resources-ovn-kubernetes-project_{context}"]
+= Listing all resources in the OVN-Kubernetes project
+
+.Prerequisites
+
+* Access to the cluster as a user with the cluster-admin role.
+* The OpenShift CLI (oc) installed.
+
+.Procedure
+
+. Run the following command to get all resources, endpoints, and configmaps in the OVN-Kubernetes project:
```

Comment:
```suggestion
* Access to the cluster as a user with the cluster-admin role.
* The OpenShift CLI (oc) installed.

.Procedure

. Run the following command to get all resources, endpoints, and config maps in the OVN-Kubernetes project:
```

Reply:
seems to be ConfigMaps in docs in other places